terms/neuro/study.json

```diff
@@ -469,6 +469,11 @@
             "description": "Multiomic Studies of Alzheimer's Disease in Adults with Down Syndrome (omicsADDS)",
             "source": "https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage?Study=syn25533707"
         },
+        {
+            "const": "Organoid_scRNAseq",
+            "description": "The AD cerebral organoid single-cell transcriptome study",
+            "source": "https://www.synapse.org/#!Synapse:syn26865580"
```

Comment:
format the source so it links to the AD portal website (just paste the synID into the end of this URL: "https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage?Study=<synID>")